simulation_sample.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import sys
+from simulations import StaticSimulation
+from simulations import DynamicSimulation
+from simulations import SimulationSet
+from geobeam import generate_route
+from geobeam import gps_utils
+
+# TODO(ameles) change this to an actual config file and a script that uses those configs to make a simulation set
+def main():
+    # a 28 minute walk
+  location1 = gps_utils.Location(37.417747, -122.086086)
+  location2 = gps_utils.Location(37.421624, -122.096472)
+  speed = 7  # meters/sec
+  frequency = 10  # Hz
+
+  user_motion = generate_route.TimedRoute(location1, location2, speed, frequency)
+  user_motion.write_route("userwalking.csv")
+  sim_one = DynamicSimulation(600, -2, "userwalking.csv")
+  sim_two = StaticSimulation(60, -2, 27.417747, -112.086086)
+  sim_three = StaticSimulation(20, -4, 37.417747, -122.086086)
+  simulation_set = SimulationSet([sim_one, sim_two, sim_three])
```

Comment:
does it make sense to add a 'addRoute' method to SimulationSets class? 

So the code would look something like:

simulation__set = SimulationSet()
simulation_set.addRoute(DynamicSimulation(600, -2, "userwalking.csv"))
simulation_set.addRoute(StaticSimulation(60, -2, 27.417747, -112.086086))
...etc

I feel it's 1) easier to read 2) eliminates the need for temp variables 

what do you think?

Reply:
I think it might add more confusion around what should happen if a simulation gets added after a run has already happened, and maybe harder on the user side to keep track of the order that they were added but i can also show the instantiation as 
```
simulation_set = SimulationSet([
DynamicSimulation(600, -2, "userwalking.csv"), 
StaticSimulation(60, -2, 27.417747, -112.086086), 
StaticSimulation(20, -4, 37.417747, -122.086086)])
```
To get rid of the temp variables

Reply:
Would a builder pattern work here?
```
simulation_set = SimulationSetBuilder()\
    .add_route(DynamicSimulation(600, -2, 'userwalking.csv'))\
    .add_route(StaticSimulation(60, -2, 27.417747, -112.086086))\
    .build()
```
`add_route` could also be split into `add_static_route` or `add_dynamic_route` to hide Simulations.